test/assert.js

```diff
@@ -778,7 +794,7 @@ describe('assert', function () {
       var errMap = new Map();
 
       errMap.set({1: 20}, 'number');
-
+   
```

Comment:
My vim does only removes it when there's an `.editorconfig` file in the root folder of where the file is 😢